Content.Client/UserInterface/Systems/Guidebook/GuidebookUIController.cs

```diff
@@ -192,6 +196,10 @@ public void OpenGuidebook(
                 RecursivelyAddChildren(guide, guides);
             }
         }
+        if (guides.ContainsKey(_lastEntry))
+        {
+            selected = _lastEntry;
+        }
```

Comment:
This will override a `selected` value if one is passed into the function, for example when pressing the species info button in the character menu

Reply:
Added a null-coalescing operator to avoid overriding. Side effect of this is that the persistence doesn't apply to handbooks and the like anymore, just the main guidebook.